src/containers/WebComponentLoader.jsx

```diff
@@ -135,6 +138,10 @@ const WebComponentLoader = (props) => {
     saveTriggered,
   });
 
+  useEffect(() => {
+    dispatch(setReactAppApiEndpoint(reactAppApiEndpoint));
+  }, [reactAppApiEndpoint, dispatch]);
```

Comment:
🔧 definitely could do with a test for this in the `WebComponentLoader.test.js` 👍